cartography/intel/kubernetes/namespaces.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import Dict
+from typing import List
+
+from neo4j import Session
+
+from cartography.intel.kubernetes.util import get_epoch
+from cartography.intel.kubernetes.util import K8sClient
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def sync_namespaces(session: Session, client: K8sClient, update_tag: int) -> Dict:
+    cluster = dict()
+    namespaces = list()
+    for namespace in client.core.list_namespace().items:
+        namespaces.append(
+            {
+                "uid": namespace.metadata.uid,
+                "name": namespace.metadata.name,
+                "creation_timestamp": get_epoch(namespace.metadata.creation_timestamp),
+                "deletion_timestamp": get_epoch(namespace.metadata.deletion_timestamp),
+            },
+        )
+        if namespace.metadata.name == "kube-system":
+            cluster = {"uid": namespace.metadata.uid, "name": client.name}
```

Comment:
Code structure issue -

Put this in a function called `get_namespaces` and call it here. The idea is to have a `sync()` function that only calls `get()` and `load()` functions. Try to refer to one of the AWS codes to get an idea of the code structure.